auth/src/test/scala/cromwell/pipeline/auth/AuthUtilsTest.scala

```diff
@@ -15,7 +15,7 @@ class AuthUtilsTest extends WordSpec with Matchers {
   private val authConfig = AuthConfig(
     secretKey = "secretKey",
     hmacAlgorithm = JwtAlgorithm.fromString(algo = "HS256").asInstanceOf[JwtHmacAlgorithm],
-    expirationTimeInSeconds = ExpirationTimeInSeconds(accessToken = 300, refreshToken = 900, userSession = 3600)
+    expirationTimeInSeconds = ExpirationTimeInSeconds(accessToken = 10000000, refreshToken = 900, userSession = 3600)
```

Comment:
return it back please ;)

Reply:
Wow, I was sure I reverted that, sorry :/